addons/packages/fluxcd-helm-controller/0.17.2/README.md

```diff
@@ -37,7 +37,7 @@ To install FluxCD helm-controller from the Tanzu Application Platform package re
    For example:
 
     ```shell
-    $ tanzu package install fluxcd-source-controller -p fluxcd-source-controller.community.tanzu.vmware.com -v 0.21.2
+    tanzu package install fluxcd-source-controller -p fluxcd-source-controller.community.tanzu.vmware.com -v 0.21.2
```

Comment:
The `0.17.2` readme should not reference the `0.21.2` version.

Reply:
It is referencing to the `fluxcd-source-controller:v0.21.2` which is compatible with the `fluxcd-helm-controller:v0.17.2` so the README is correct in that sense